README.md

```diff
@@ -1,22 +1,24 @@
 [![PyPI version](https://img.shields.io/pypi/v/dicom_parser.svg)](https://pypi.python.org/pypi/pylabber/)
 [![PyPI status](https://img.shields.io/pypi/status/dicom_parser.svg)](https://pypi.python.org/pypi/pylabber/)
 [![GitHub Actions](https://github.com/ZviBaratz/dicom_parser/actions/workflows/tests.yml/badge.svg)](https://github.com/ZviBaratz/dicom_parser/actions/workflows/tests.yml)
+[![Documentation Status](https://readthedocs.org/projects/dicom-parser/badge/?version=latest)](http://dicom-parser.readthedocs.io/?badge=latest)
+
 [![codecov.io](https://codecov.io/gh/ZviBaratz/dicom_parser/coverage.svg?branch=master)](https://codecov.io/github/ZviBaratz/dicom_parser?branch=master)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/ZviBaratz/dicom_parser.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/ZviBaratz/dicom_parser/context:python)
-[![Documentation Status](https://readthedocs.org/projects/dicom-parser/badge/?version=latest)](http://dicom-parser.readthedocs.io/?badge=latest)
+[![CodeFactor](https://www.codefactor.io/repository/github/zvibaratz/dicom_parser/badge)](https://www.codefactor.io/repository/github/zvibaratz/dicom_parser)
 
 # dicom_parser
 
 _dicom_parser_ is a utility python package meant to facilitate access to
-[DICOM](https://www.dicomstandard.org/) header information by extending the functionality of _[pydicom]_.
+[DICOM](https://www.dicomstandard.org/) header information by extending the functionality of _[pydicom][]_.
```

Comment:
Codacy has a fix for the issue: [Spaces inside emphasis markers](https://app.codacy.com/gh/ZviBaratz/dicom_parser/pullRequest?prid=7629233)
```suggestion
[DICOM](https://www.dicomstandard.org/) header information by extending the functionality of_[pydicom][]_.
```